salt/utils/platform.py

```diff
@@ -209,3 +210,12 @@ def is_aarch64():
     Simple function to return if host is AArch64 or not
     """
     return platform.machine().startswith("aarch64")
+
+
+def spawning_platform():
```

Comment:
**question** is it better to gate this behind a function or do something like this at the module level?

```
SPAWNING_PLATFORM = multiprocessing.get_start_method(allow_none=False) == "spawn"
```

I don't know if it's called enough to make a difference :shrug:

Reply:
No, the method can be changed programmatically at run time. A global variable would only pick the initial config value.

Reply:
@waynew are we good to resolve this review comment?

Reply:
Also, if you notice, this is probably the only function which doesnt have the cached decorator, that was the reason behind it.